website/docs/reference/deprecations.md

```diff
@@ -241,3 +300,39 @@ information: https://docs.getdbt.com/reference/global-configs/legacy-behaviors
 #### SourceFreshnessProjectHooksNotRun warning resolution
 
 Set `source_freshness_run_project_hooks` to `true`. For instructions on skipping project hooks during a `dbt source freshness` invocation, check out the [behavior change documentation](/reference/global-configs/behavior-changes#project-hooks-with-source-freshness).
+
+### UnexpectedJinjaBlockDeprecation
+
+If you have an unexpected Jinja block - an orphaned Jinja block or a Jinja block outside of a macro context - you will receive a warning, and in a future version, dbt will stop supporting unexpected Jinja blocks. Previously, these unexpected Jinja blocks were silently ignored.
+
+<File name='macros/my_macro.sql'>
+
+```sql
+
+{% endmacro %} # orphaned endmacro jinja block
+
+{% macro hello() %}
+hello!
+{% endmacro %}
+
+```
+</File>
+
+If you have an unexpected Jinja block, you will receive a warning, and in a future version, dbt will stop supporting unexpected Jinja blocks. Previously, these unexpected Jinja blocks were silently ignored.
```

Comment:
This repeats the above, right? I think we can delete [lines 321 to 334](https://github.com/dbt-labs/docs.getdbt.com/pull/7329/files#diff-0e4a2f7e2e1b9d4f984c9833dcb896edd5dd9cd91898de0567da4d35cdbe8e7aR321-R334)

Reply:
Whoops! Good catch